src/vcastAdapter.ts

```diff
@@ -590,7 +590,7 @@ export async function executeTest(
   } else {
     commandStatus = executeTestViaPython(vcastCommand, enviroPath, nodeID);
   }
-
+  vectorMessage(`COMMAND STATUS: ${JSON.stringify(commandStatus, null, 2)}`);
```

Comment:
What's this print about?

Reply:
Was for debugging to get the exitCode. Deleted in 3509429a2b77635f18359b6f01833aa8409d9dd9.